jsmn-find.h

```diff
@@ -161,52 +159,61 @@ JSMN_API long jsmnf_unescape(char buf[],
 #include <stdlib.h>
 #include <string.h>
 
-/* key */
-#define CHASH_KEY_FIELD     k
-/* value */
-#define CHASH_VALUE_FIELD   v
-/* fields */
-#define CHASH_BUCKETS_FIELD fields
-/* members count */
-#define CHASH_LENGTH_FIELD  size
-
-#include "chash.h"
+/* hashtable */
+#define TABLEC_HT struct jsmnf_pair
```

Comment:
I think we need a better prefix here, such as `TABLEC_SYMBOL_`

Reply:
Done